types/message.go

```diff
@@ -8,8 +8,8 @@ type Message struct {
 	Nonce      uint64        `json:"nonce,omitempty"`
 	From       string        `json:"from,omitempty"`
 	To         string        `json:"to,omitempty"`
-	Value      big.Int       `json:"value,omitempty"`    // in AttoFIL
-	GasPrice   big.Int       `json:"gasPrice,omitempty"` // in AttoFIL
+	Value      *big.Int      `json:"value,omitempty"`    // in AttoFIL
+	GasPrice   *big.Int      `json:"gasPrice,omitempty"` // in AttoFIL
```

Comment:
👍 I noticed [this issue](https://github.com/golang/go/issues/28946#issuecomment-441684687) when marshalling pointers to `big.Int` also.